{"name": "Проверка создания пользователя, который уже зарегистрирован", "status": "passed", "description": "Запрос возвращает код ответа 403 и ответ {\"success\": False, \"message\": \"User already exists\"}", "start": 1715273489645, "stop": 1715273489905, "uuid": "cc70ff41-f64f-4e51-9dda-30629fccf547", "historyId": "09ed417780fcc92a2702f89a9707f08c", "testCaseId": "09ed417780fcc92a2702f89a9707f08c", "fullName": "tests.test_create_user.TestCreateUser#test_create_user_duplicate", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_user"}, {"name": "subSuite", "value": "TestCreateUser"}, {"name": "host", "value": "MacBook-Air-Oksana.local"}, {"name": "thread", "value": "38372-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_user"}]}